src/components/Inventory/withdraw/BatchWithdrawFlow.tsx

```diff
@@ -118,11 +118,36 @@ export default function BatchWithdrawFlow(props: BatchWithdrawFlowProps): JSX.El
     }
 
     // set snackbar with status
-    snackbar.toastSuccess(strings.SAVE_CHANGES); // TODO set status as per design
+    snackbar.toastSuccess(getFormattedSuccessMessage(withdrawal as NurseryWithdrawal));
     // redirect to inventory
     goToInventory();
   };
 
+  const getFormattedSuccessMessage = (withdrawal: NurseryWithdrawal): string => {
+    const numBatches = withdrawal.batchWithdrawals?.length;
+    const totalWithdrawn = withdrawal.batchWithdrawals?.reduce((total, batchWithdrawal) => {
+      const { germinatingQuantityWithdrawn, notReadyQuantityWithdrawn, readyQuantityWithdrawn } = batchWithdrawal;
+      if (germinatingQuantityWithdrawn) {
+        total += germinatingQuantityWithdrawn;
+      }
+      if (notReadyQuantityWithdrawn) {
+        total += notReadyQuantityWithdrawn;
+      }
+      if (readyQuantityWithdrawn) {
+        total += readyQuantityWithdrawn;
+      }
+      return total;
+    }, 0);
+
+    return strings.formatString(
+      strings.BATCH_WITHDRAW_SUCCESS,
+      numBatches,
+      numBatches === 1 ? strings.BATCHES_SINGULAR : (strings.BATCHES_PLURAL as any),
```

Comment:
why is that “as any” needed?

Reply:
Typescript was expecting formatted number object